t/APISIX.pm

```diff
@@ -96,6 +96,18 @@ etcd:
 _EOC_
 }
 
+my $custom_hmac_auth = $ENV{"CUSTOM_HMAC_AUTH"} || "false";
+if ($custom_hmac_auth eq "true") {
+    $user_yaml_config .= <<_EOC_;
+plugin_attr:
+  hmac-auth:
+    signature_key: "X-APISIX-HMAC-SIGNATURE"
```

Comment:
we can remove `"` here, make is simpler.

eg: `signature_key: X-APISIX-HMAC-SIGNATURE`